### PR TITLE
Make repo compatible with PRBT

### PR DESCRIPTION
**Pull request by Chamango90**
Depends on https://github.com/rosin-project/automatica18_scan_and_plan_demo/pull/2.

Simulation is working if: 

- [x] `<inertial>` block is added to `base_link` ([ref](https://github.com/PilzDE/pilz_robots/blob/kinetic-devel/prbt_support/urdf/prbt_macro.xacro#L35))

Further improvements:

- [x] Substitute psir in own packages / files with prbt

### CODE REVIEW

File: psir_demo_moveit_config/config/controllers.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 controller_list:
-  - name: "/psir/arm_joint_trajectory_controller"
+  - name: "/prbt/manipulator_joint_trajectory_controller"
```
Comment (hsd-dev) [inline psir_demo_moveit_config/config/controllers.yaml]: shouldn't this be `/prbt/arm_joint_trajectory_controller`?

File: snp_psir_bringup/launch/psir_real.launch [hunk 2]
```diff
@@ -1,69 +1,25 @@
...
-  <param name="robot_description" type="str" command="$(find xacro)/xacro --inorder '$(arg urdf_file)' sim:=false"/>
-
-  <rosparam file="$(find snp_psir_bringup)/config/arm_controller.yaml" command="load" />
-  <!--node name="controller_spawner" pkg="controller_manager" type="spawner" respawn="false" output="screen"
-    ns="/$(arg robot_name)" args="joint_state_controller joint_trajectory_controller" /-->
-
 	<!-- run the "real robot" interface nodes -->
-  <!--   - this typically includes: robot_state, motion_interface, and joint_trajectory_action nodes -->
-  <!--   - replace these calls with appropriate robot-specific calls or launch files -->
     <include file="$(find psir_description)/launch/robot.launch" >
```
Comment (hsd-dev) [inline snp_psir_bringup/launch/psir_real.launch]: Maybe `psir_description` has been replaced by `prbt_support`?

File: psir_demo_moveit_config/config/controllers.yaml (hunk 1 above)
Comment (Chamango90) [reply psir_demo_moveit_config/config/controllers.yaml]: The controller group name has changed

File: snp_psir_bringup/launch/psir_real.launch (hunk 2 above)
Comment (Chamango90) [reply snp_psir_bringup/launch/psir_real.launch]: Yes you are right. This is a TODO. I looked for now only into the simulation.

**Comment by hsd-dev** [comment]
The PR works for me :+1:

**Comment by Chamango90** [comment]
PR is finished now.

Except:

> 
> Simulation is working if:
> inertial block is added to base_link (ref)

**Comment by Chamango90** [comment]
PR is now final. ~Compilation and simulation is working with https://github.com/PilzDE/pilz_robots/pull/17.~
Update: PR merged.

**Pull request merged**